src/chrome/content/rules/Onlyoffice.org.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+	For other Onlyoffice coverage, see Onlyoffice.com.xml.
+
+	Expired:
+		- onlyoffice.org
+		- www.onlyoffice.org
+
+	Host has wildcard DNS.
+-->
+<ruleset name="Onlyoffice.org">
+	<target host="onlyoffice.org" />
```

Comment:
https://travis-ci.org/EFForg/https-everywhere/jobs/586533856#L327

Reply:
Let's see if this happens again...

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/586653806#L327
again

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/592913801#L334
again and again.

Reply:
I guess remove these targets, mark them as "SSL error" and mark the ruleset as "partial".